main/test/ca/uwaterloo/flix/tools/TestPackager.scala

```diff
@@ -40,6 +44,21 @@ class TestPackager extends FunSuite {
     assert(jarPath.getFileName.toString.startsWith(ProjectPrefix))
   }
 
+  test("build-jar generates ZIP entries with fixed time") {
```

Comment:
The idea is that the end result of two different builds with the same options is byte-for-byte exactly the same, right? So would it make sense to have a test where we build two separate builds and compare the output files?